src/main/java/com/google/sps/servlets/utils/CommentAnalysis.java

```diff
@@ -53,31 +53,56 @@ public CommentAnalysis(LanguageServiceClient languageService) {
    * @return a Statistics object that contains required values to display
    */
   public Statistics computeOverallStats(CommentThreadListResponse youtubeResponse) {
-    List<Double> scoreValues =
+    // all sentiments built by comment content retrieved from youtubeResponse
+    List<Sentiment> documentList =
         youtubeResponse.getItems().stream()
             .map(UserComment::new)
-            .map(this::calcualateSentiAnalysisScore)
+            .map(
+                comment ->
+                    languageService
+                        .analyzeSentiment(
+                            Document.newBuilder()
+                                .setContent(comment.getCommentMsg())
+                                .setType(Document.Type.PLAIN_TEXT)
```

Comment:
Can you put this logic into its own method?

This is large enough that it should be qualified with a method, maybe `calculateSentimentForComment(UserComment userComment)`?